installer/Migration/V4_7_0/Migration.php

```diff
@@ -28,123 +28,7 @@ class Migration extends AbstractMigration
-                ->addColumn('value', Types::TEXT)
-                ->addColumn('translated', Types::SMALLINT, ['Unsigned' => true, 'Default' => 1])
-                ->addColumn('customized', Types::SMALLINT, ['Unsigned' => true, 'Default' => 0])
-                ->addColumn('version', Types::STRING, ['Length' => 20, 'Default' => null, 'Notnull' => false])
-                ->addColumn(
-                    'modified_at',
-                    Types::DATETIMETZ_MUTABLE,
-                    ['Notnull' => false, 'Default' => 'CURRENT_TIMESTAMP']
-                )
-                ->setPrimaryKey(['id'])
-                ->create();
-        }
-
-        if (!$this->getSchemaHelper()->tableExists(['ohrm_i18n_source'])) {
-            $this->getSchemaHelper()->createTable('ohrm_i18n_source')
-                ->addColumn('id', Types::INTEGER, ['Autoincrement' => true])
-                ->addColumn('source', Types::TEXT, ['Length' => 18, 'Notnull' => false])
-                ->addColumn('modified_at', Types::DATETIME_MUTABLE, ['Notnull' => false,])
-                ->setPrimaryKey(['id'])
-                ->create();
-        }
-
-        $foreignKeyConstraint = new ForeignKeyConstraint(
-            ['group_id'],
-            'ohrm_i18n_group',
-            ['id'],
-            'groupId',
-            ['onDelete' => 'SET NULL']
-        );
-        $this->getSchemaHelper()->addForeignKey('ohrm_i18n_lang_string', $foreignKeyConstraint);
-
-        $foreignKeyConstraint = new ForeignKeyConstraint(
-            ['language_id'],
-            'ohrm_i18n_language',
-            ['id'],
-            'languageId',
-        );
-        $this->getSchemaHelper()->addForeignKey('ohrm_i18n_translate', $foreignKeyConstraint);
-
-        $foreignKeyConstraint = new ForeignKeyConstraint(
-            ['lang_string_id'],
-            'ohrm_i18n_lang_string',
-            ['id'],
-            'langStringId',
-        );
-        $this->getSchemaHelper()->addForeignKey('ohrm_i18n_translate', $foreignKeyConstraint);
-
-        $foreignKeyConstraint = new ForeignKeyConstraint(
-            ['source_id'],
-            'ohrm_i18n_source',
-            ['id'],
-            'sourceId',
-        );
-        $this->getSchemaHelper()->addForeignKey('ohrm_i18n_lang_string', $foreignKeyConstraint);
-
-        $uniqueConstraint = new UniqueConstraint(
-            'translateUniqueId',
-            ['lang_string_id', 'language_id']
-        );
-        $this->getSchemaHelper()->getSchemaManager()->createUniqueConstraint($uniqueConstraint, 'ohrm_i18n_translate');
-        $this->getDataGroupHelper()->insertScreenPermissions(__DIR__ . '/permission/screen.yaml');
-        $languages = $this->readlanguageYaml(__DIR__ . '/language/languages.yaml');
-        $this->insertLanguages($languages);
-        $groups = $this->readlanguageYaml(__DIR__ . '/language/groups.yaml');
-        $this->insertGroups($groups);
+        // no db chnages for this version
```

Comment:
double check all the migrations files, to confirm whether this kind of mistakes exists

Reply:
Checked